llvm/test/DebugInfo/symbolize-gnu-debuglink-no-realpath.test

```diff
@@ -1,4 +1,6 @@
-# REQUIRES: shell
+# This test uses symlinks.
+UNSUPPORTED: system-windows
```

Comment:
I think we need the leading `#`?

Reply:
No, since this file is _only_ fed into FileCheck, no `#` is needed either for the comments or for the FileCheck directives. I just copied the style used below.